kube/minkube/services/ssh-service.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 KernelCI Authors.
+# Author: Sanskar Bhushan <sbdtu5498@gmail.com>
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: kernelci-api-ssh
+spec:
+  selector:
+    app: kernelci-api-ssh
+  ports:
+    - protocol: TCP
+      port: 22
```

Comment:
```suggestion
      port: 8022
```

Reply:
I think using the standard ports makes sense i.e. 22 for SSH, then when exposing the port to the public it can be a different one depending on the deployment specifics

Reply:
This is to expose the container port for the pipeline.